nucypher/blockchain/eth/token.py

```diff
@@ -249,43 +269,56 @@ def address_index_ordering_key(self):
     # Metadata
     #
 
-    @property
-    def is_expired(self) -> bool:
-        current_period = self.staking_agent.get_current_period()  # TODO #1514 this is online only.
-        return bool(current_period > self.final_locked_period)
+    def status(self, staker_info: StakerInfo = None, current_period: Period = None) -> Status:
+        if self._status:
```

Comment:
Perhaps a docstring here describing the status tree.

Reply:
✔️